backend/de.metas.util/src/main/java/de/metas/util/time/DurationUtils.java

```diff
@@ -75,4 +100,25 @@ else if (unit == ChronoUnit.DAYS)
 		}
 	}
 
+	private static BigDecimal getEquivalentInSmallerTemporalUnit(@NonNull final BigDecimal durationBD, @NonNull final TemporalUnit unit)
+	{
+		if (unit == ChronoUnit.DAYS)
+		{
+			return durationBD.multiply(BigDecimal.valueOf(8));// This refers to work hours, not calendar hours
```

Comment:
why work hours and not simply hours?

Reply:
i.e. multiply by 24.

Reply:
These calculations are used in a work hours context, that's why I also changed the method name: `toDuration` -> `toWorkDuration`.